website/docs/how-to/how-to-setup-provisioning-with-entra.md

```diff
@@ -81,17 +83,11 @@ You should remove all unnecessary properties. This ensures that Entra will reach
 - emails
 - externalId
 
-**4) Update the active property**
-
-You need to set the active property to the following (lowercase the boolean values):
-
-Switch([IsSoftDeleted], , "false", "true", "true", "false")
-
-**5) Update the email property to **
+**4) Update the email property to**
```

Comment:
This seems to be missing the end of the sentence. Update the email property to what? 

... I realize it also didn't say before, though 🤷🏼

Reply:
You're absolutely correct, that should finish with "userPrincipleName", fixed now